<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemInfoDialog</class>
 <widget class="QDialog" name="ItemInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_leave">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Оставить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Open">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть кейс
265 руб.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Sold">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Продать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Photo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>280</width>
     <height>217</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Неизвестно</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelStatic_Cost">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Это стоит:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Cost">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>? руб.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelStatic_Float">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>float</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Float">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Float_Name">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>146</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>pushButton_leave</zorder>
  <zorder>pushButton_Open</zorder>
  <zorder>pushButton_Sold</zorder>
  <zorder>label_Photo</zorder>
  <zorder>label_Name</zorder>
  <zorder>label_Cost</zorder>
  <zorder>labelStatic_Float</zorder>
  <zorder>label_Float</zorder>
  <zorder>label_Float_Name</zorder>
  <zorder>labelStatic_Cost</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_leave</sender>
   <signal>clicked()</signal>
   <receiver>ItemInfoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
